### PR TITLE
Restructured tools and integrations section

### PR DESCRIPTION
**Pull request by sanderson**
This PR restructures the "Tools" section into "Tools & integrations" and moves the following docs into the updated section:

- Client libraries
- InfluxDB CLIs (landing page only)
- Grafana
- Postman

image

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/v2.0/tools/clis.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  Use the `influx` and `indfluxd` command line interfaces to interact with and
+  manage InfluxDB.
+menu:
+  influxdb_2_0:
+    parent: Tools & integrations
+weight: 102
+influxdb/v2.0/tags: [cli]
+---
+
+Use the `influx` and `indfluxd` command line interfaces (CLIs) to interact with and
```
Comment (pierwill) [inline content/influxdb/v2.0/tools/clis.md:13]: `influxd`

File: content/influxdb/v2.0/tools/clis.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+manage InfluxDB.
+
+## influx CLI
+Use the `influx` CLI to interact with and manage both **InfluxDB Cloud** and **InfluxDB OSS**.
+Write and query data, generate InfluxDB templates, export data, manage organizations
+and users, and more.
+
+_For more information, see the [`influx` reference documentation](/influxdb/v2.0/reference/cli/influx/)._
+
+## influxd CLI
```
Comment (pierwill) [inline content/influxdb/v2.0/tools/clis.md:23]:
```
## `influxd` CLI
```

**Review by pierwill — APPROVED** [review]
Looks good! 👍

File: content/influxdb/v2.0/tools/clis.md (hunk 1 above)
Comment (sanderson) [reply content/influxdb/v2.0/tools/clis.md:13]: Nice catch, thx!

File: content/influxdb/v2.0/tools/clis.md (hunk 2 above)
Comment (sanderson) [reply content/influxdb/v2.0/tools/clis.md:23]: We override/don't allow `<code>` styling in headings, so this won't render any differently. Adding the inline code will just add unnecessary html markup.

File: content/influxdb/v2.0/tools/clis.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+---
+title: InfluxDB CLIs
```
Comment (kelseiv) [inline content/influxdb/v2.0/tools/clis.md:2]: How about "Use InfluxDB CLIs" so it feels more procedural than reference?

Then, we can add procedures/examples for how to "Write and query data", "Generate InfluxDB templates", "Export data", "Manage organizations" moving forward

**Review by kelseiv — APPROVED** [review]

File: content/influxdb/v2.0/tools/clis.md (hunk 3 above)
Comment (sanderson) [reply content/influxdb/v2.0/tools/clis.md]: Updated. The nav now looks like this:

image

**Pull request merged**